types/benchmark/index.d.ts

```diff
@@ -135,6 +135,40 @@ declare namespace Benchmark {
         elapsed: number;
         timeStamp: number;
     }
+    
+    export interface Target {
+        options: Options;
+        async?: boolean;
+        defer?: boolean;
+        delay?: number;
+        initCount?: number;
+        maxTime?: number;
+        minSamples?: number;
+        minTime?: number;
+        name?: string;
+        fn?: Function;
+        id: number;
+        stats?: {
+            moe?: number;
+            rme?: number;
+            sem?: number;
+            deviation?: number;
+            mean?: number;
+            sample?: any[];
+            variances?: number;
```

Comment:
If this block isn't replaced with Stats, then `variances` should be `variance`, the word `variances` doesn't appear in benchmark's source code.